src/content/reference/fleet-management/clusters/introduction_cluster_api.md

```diff
@@ -0,0 +1,45 @@
+
+## Giant Swarm flavor
+
+The Giant Swarm platform offers additional capabilities on top of Cluster API that we consider as must-have for a complete production system and those are not part of the upstream project, yet.
+
+We leverage these features of Cluster API:
+
+- Certificate management: Cluster API manages important certificates in a Kubernetes cluster, like for kubelet or etcd.
+- Node bootstrap: the operator correctly bootstraps nodes in the cluster from the defined configuration.
+- Infrastructure provisioning: the operator talks to the provider-specfic API (e.g. AWS API) to create the infrastructure needed for the cluster.
+- Cluster autoscaling: Cluster API integrates with the [cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) project to allow clusters to grow or shrink based on different policies.
+- Health checks: the operator is able to check the health of the nodes and take actions to avoid manual intervention.
+
+On top of that, we have some additional features that are not part of the Cluster API standard:
+
+- DNS management: we provide a DNS operator to manage DNS assets for the clusters dynamically.
+- Provider credentials management: we have automated the credentials management to operate the provider infrastructure in a secure way.
+- Encryption configuration: we have designed an operator that controls the cryptographic assets for workload clusters and configure them properly making sure they are rotated and updated as needed.
+- Cluster API monitoring: we monitor all the cluster API specific resources letting us know the status of the cluster creation and the health of the cluster.
+- Further, we have some other operators that are provider specific and helps to automate all parts of the cluster lifecycle, like VPC provisioning or security group management, when upstream Cluster API does not provide it.
+
+## Wrapping resources in apps
+
+For creating a cluster using Cluster API, you would need to define a set of resources that will be used to create the cluster. For example: `Cluster`, `AWSCluster`, `KubeadmControlPlane`, `MachinePool`, `AWSMachinePool`, etc. as standardized by the Cluster API custom resource definitions. You _could_ create these resources manually, exploring every single specification field that is possible to use, but we offer a more convenient way to do that: we have a set of apps that are ready to be used to provision a cluster correctly. For those, we provide proper release management, documentation and testing, such that you can trust that minor upgrades go smoothly, or breaking changes are described and happening only in major releases.
+
+An [`App` is a custom resource]({{< relref "vintage/getting-started/app-platform/app-bundle" >}}) which helps us to group all needed resources allowing exposure to the configuration in a way that can be validated, defaulted and documented properly.
```

Comment:
Isn't it more right to say we have a helm chart for it? The app CR is just our special interface to install that helm chart declaratively, but what we created and are managing releases and documentation with is the chart.

The reason I bring this up is that if you call it app it looks proprietary when it is in fact something open that can be used with anything that supports helm. In the future we might just use HelmReleases instead of app CRs, calling it app in that context would be confusing IMHO

Reply:
hmm not 100% agree. IMO chart CR will most likely be replaced with HelmRelease but the app CR still needs to be there for the MC mgmt. Anyway, I will also refer to the helm chart concept so don't seem too proprietary (I got your point) but not mentioning it right now IMO it is confusing; if the customer follows the link to create a cluster they will get the kgs command which generates an app CR (not a helm chart).

Reply:
PTAL, I did some modifications

Reply:
Yeah, that works, my main point was to mention that it's not proprietary (i.e. a helm chart, not chart cr) and I like how you then make it clear that app CR extends usage of helm charts.